### PR TITLE
Migrated backpack custom rules to backstrap

### PR DESCRIPTION
**Pull request by promatik**
This PR is an update of all the rules contained on backpack custom css files; `list.css`, `form.css` and `crud.css`.

### CODE REVIEW

**Comment by iMokhles** [comment]
for RTL padding-left: 0.8rem; won't work as expected :(

**Comment by promatik** [comment]
Hi @iMokhles, thank you for the advise!
I'll fix that 👌

File: src/scss/customs/_crud.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.sidebar .nav-dropdown-items .nav-dropdown {
+    padding-left: 0.8rem;
+
+    &:not(.open) > a {
+        font-weight: normal!important;
+    }
+}
```
Comment (promatik) [inline src/scss/customs/_crud.scss]:
```suggestion
.sidebar .nav-dropdown-items .nav-dropdown {
    padding: 0 0 0 .8rem;

    &:not(.open) > a {
        font-weight: normal!important;
    }
}

[dir="rtl"] .sidebar .nav-dropdown-items .nav-dropdown {
    padding: 0 .8rem 0 0;
}
```

**Comment by promatik** [comment]
@iMokhles thank you for the advice 🙌

It's fixed now;

Left to Right;
image

Right to Left;
image

**Comment by tabacitu** [comment]
Merged & tagged as 0.4.0 - 🤞

**Pull request merged**